docs/modules/quick-start/pages/situations.adoc

```diff
@@ -0,0 +1,97 @@
+
+=== Manage situation
+
+You can take the following actions on a situation from its details page:
+
+* *Acknowledge:* Indicate that the situation has been seen and is being addressed.
+Your account will take ownership of the situation within OpenNMS.
+All associated alarms will also be acknowledged, and your account will take ownership of them.
+* *Clear:* Indicate that no action needs to be taken and remove the situation from the list of active situations.
+All associated alarms will also be cleared.
+* *Reject:* Indicate that the alarms in the situation are not related.
+
+You can also maintain sticky memos and journal memos for the situation.
+Sticky memos are used to note key information for the situation overall (for example, the potential root cause).
+Journal memos are used to log any important work or investigation steps; this can include root cause analysis or developing a fix for the underlying issue.
+
+Follow these steps to create a memo:
+
+. Navigate to a situation's details page.
+. Click the *pencil* symbol beside *Sticky Memo* or *Journal Memo*, as desired.
+. Type your note in the text box that is displayed.
+. Click the *green check* symbol to save the memo.
+
+=== Manage alarms
+
+You can take the following actions on individual alarms associated with a situation:
+
+* *Clear:* Indicate that no action needs to be taken and remove the alarm from the list of active alarms.
+* *Acknowledge:* Indicate that the alarm has been seen and is being addressed.
+Your account will take ownership of the alarm within OpenNMS.
+* *Move:* Move the alarm to a different situation.
+This should be used when another situation more accurately encompasses an issue.
+* *Remove:* Remove the alarm from the situation.
+This should be used when an alarm has been incorrectly associated with a situation.
+
+Follow these steps to manage individual alarms:
+
+. Navigate to a situation's details page.
+. Select an alarm from the *Alarms* list.
+. Evaluate the alarm and select the appropriate action (for example, acknowlege an alarm that you will be working on).
+. (Optional) Create a sticky memo or a journal memo.
+** Click the *pencil* symbol beside *Sticky Memo* or *Journal Memo*, as desired.
+** Type your note in the text box that is displayed.
+** Click the *green check* symbol to save the memo.
+
+=== View situation timeline
+
+To view a timeline of the situation and its associated alarms, navigate to the *Metrics* tab on the details page.
+The timeline displays a graphical overview of alarm activity throughout the situation's lifetime.
+
+.Situation timeline page
+image::situation-timeline.png[Situation timeline displaying three active alarms, 800]
+
+== Create a situation
+
+You may occasionally notice alarms that are related, but that were not grouped into a situation.
+In these cases, you can create a situation to represent such relationships:
+
```

Comment:
```suggestion
NOTE: You can also view and move unassociated alarms into existing situations.
(See xref: .... [Manage unassociated alarms].)
```
Suggest adding a topic below this "Manage unassociated alarms" that describes the process of clicking the "View Unassociated Alarms" button and then moving them to a situation. We have described the "move" feature as part of alarm management above, but it would be good to show them this route that is specific to unassociated alarms. We also need to check with Ben or Gerardo if unassociated alarms *must* be in a situation in order to manage them. I don't see a way to clear, acknowledge, or reject them from that screen.